src/test/kotlin/com/forgerock/sapi/gateway/ob/uk/tests/functional/payment/domestic/payments/consents/api/v3_1_8/GetDomesticPaymentsConsents.kt

```diff
@@ -16,52 +15,32 @@ class GetDomesticPaymentsConsents(val version: OBVersion, val tppResource: Creat
     private val paymentLinks = getPaymentsApiLinks(version)
-    
+
     fun shouldGetDomesticPaymentsConsents() {
         // Given
         val consentRequest = OBWriteDomesticConsentTestDataFactory.aValidOBWriteDomesticConsent4()
-        val consent = createDomesticPaymentsConsentsApi.createDomesticPaymentsConsent(consentRequest)
-
-        assertThat(consent).isNotNull()
-        assertThat(consent.data).isNotNull()
-        assertThat(consent.data.consentId).isNotEmpty()
-        Assertions.assertThat(consent.data.status.toString()).`is`(Status.consentCondition)
-        assertThat(consent.risk).isNotNull()
-
         // When
-        val result = createDomesticPaymentsConsentsApi.getPatchedConsent(consent)
-
+        val consentResponse = createDomesticPaymentsConsentsApi.createDomesticPaymentsConsent(consentRequest)
         // Then
-        assertThat(result).isNotNull()
-        assertThat(result.data).isNotNull()
-        assertThat(result.risk).isNotNull()
-        assertThat(result.data).isEqualTo(consent.data)
-        assertThat(result.risk).isEqualTo(consent.risk)
+        assertThat(consentResponse).isNotNull()
+        assertThat(consentResponse.data).isNotNull()
+        assertThat(consentResponse.data.consentId).isNotEmpty()
+        Assertions.assertThat(consentResponse.data.status.toString()).`is`(Status.consentCondition)
+        assertThat(consentResponse.risk).isNotNull()
     }
 
     fun shouldGetDomesticPaymentsConsents_withoutOptionalDebtorAccountTest() {
         // Given
         val consentRequest = OBWriteDomesticConsentTestDataFactory.aValidOBWriteDomesticConsent4()
         consentRequest.data.initiation.debtorAccount(null)
-
-        val consent = createDomesticPaymentsConsentsApi.createDomesticPaymentsConsent(consentRequest)
-
-        assertThat(consent).isNotNull()
-        assertThat(consent.data).isNotNull()
-        assertThat(consent.data.consentId).isNotEmpty()
-        Assertions.assertThat(consent.data.status.toString()).`is`(Status.consentCondition)
-        assertThat(consent.risk).isNotNull()
-        assertThat(consent.data.initiation.debtorAccount).isNull()
-
-        // When
-        val result = createDomesticPaymentsConsentsApi.getPatchedConsent(consent)
-
+        // when
+        val consentResponse = createDomesticPaymentsConsentsApi.createDomesticPaymentsConsent(consentRequest)
         // Then
-        assertThat(result).isNotNull()
-        assertThat(result.data).isNotNull()
-        assertThat(result.risk).isNotNull()
-        assertThat(result.data).isEqualTo(consent.data)
-        assertThat(result.risk).isEqualTo(consent.risk)
-        assertThat(result.data.initiation.debtorAccount).isNull()
+        assertThat(consentResponse).isNotNull()
+        assertThat(consentResponse.data).isNotNull()
+        assertThat(consentResponse.data.consentId).isNotEmpty()
+        Assertions.assertThat(consentResponse.data.status.toString()).`is`(Status.consentCondition)
```

Comment:
It's a different assertion class used, from assertj instead of assertk, at the moment it's better don't optimise the imports, we need to optimise the libraries used to assertions.
`assertThat(consentResponse).isNotNull()` it's different method from different class of `Assertions.assertThat(consentResponse.data.status.toString())`